_posts/2024-03-22-Civic-Tech-Career-Resources.md

```diff
@@ -0,0 +1,187 @@
+
+[San Francisco Digital Services](https://digitalservices.sfgov.org/)
+
+[Colorado Digital Service](https://oit.colorado.gov/colorado-digital-service)
+
+
+## Canada
+
+Don’t forget to check out our friends up north!
+[Ontario Digital Services](https://www.ontario.ca/page/ontario-digital-service)
+
+## Government Jobs Boards
+
+Some of the great programming positions aren’t tied to big, new, splashy, digital
+services teams; they often get syndicated on government career websites.
+Some of these agencies are getting better at having their open positions posted
+more broadly on sites like LinkedIn. As these jobs aren’t distributed widely,
+sometimes they don’t get many applicants. These positions can also have
+less obvious titles. For example, in the County of Los Angeles, programmers often
+have the title Business Analyst. We suggest you broadly browse these job boards
+and notice what kinds of titles are used to describe the skills you have.
+
+Some of these jobs require you to pass an “exam” or “test” -- don’t stress too
+much; do a bit of research. Sometimes these are standardized, multiple choice
+exams you take with a pencil in a proctored room, while others are just a long-form
+written version of your resume submitted via an online form. Do note that if
+the position requires you to have passed an exam, you will need to do so before
+you submit your application or they will auto-reject your application.
+
+[State of CA: CalCareers](https://www.calcareers.ca.gov/): This site has so many jobs! You can create an account and get really specific email
+notifications sent to you when a position opens that you might be qualified for or
+within an agency you might be excited about working with.
+  - CalCareers is confusing to learn how to navigate – [ODI has a really good guide](https://innovation.ca.gov/join-us/applying-civil-service-job/ ) to the state application process.
+
+[Government Jobs](https://www.governmentjobs.com/):
+Private syndicate of gov jobs.
+
+[USA Jobs](https://www.usajobs.gov/):
+Syndicate of most federal jobs.
+
+## Public Transit Agencies
+**US Department of Transportation**:
+This [USAjobs.gov link filters for jobs at DOT](https://dot.usajobs.gov/search/results/?d=TD&p=1)
+
+**Federal Transit Administration** *(“FTA is one of 10 modal agencies of the US Department
+of Transportation”)*
+- Find jobs by searching USA Jobs
+- Sign up for email notifications via the [Transportation Diversity Council’s (TDC)](https://tdc-ntl.org/blog/category/job-postings/) website.
+- Check out the [FTA’s job website](https://www.transit.dot.gov/jobs) for more information on applying for jobs and links to early career programs
+- [FTA has several early career programs](https://www.transit.dot.gov/about/jobs/early-career-programs)
+
+## **And of course, last but not least, [sign up for notifications to hear when Compiler is hiring again.](https://compiler.us14.list-manage.com/subscribe?u=83dad32003c2669592d5c0b95&id=c5d6b636b2)**
```

Comment:
Few things here:

* This shouldn't be a heading (the `##` text makes it a heading 2)
* This is the link to the old Mailchimp list, should be pointing to the new Hubspot list: https://share.hsforms.com/1yPW2aSteQS6VTeGDEyFq9gqq0e7
* I feel like the text `last but not least` is a little unnecessary (IMHO)

Reply:
@thekaveman thanks, made the edits.